parsec/core/cli/human_find.py

```diff
@@ -0,0 +1,36 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2016-2021 Scille SAS
+
+import click
+
+from parsec.utils import trio_run
+from parsec.core import logged_core_factory
+from parsec.core.cli.utils import cli_command_base_options, core_config_and_device_options
+from parsec.cli_utils import cli_exception_handler
+
+
+async def _human_find(
+    config,
+    device,
+    query,
+    page: int = 1,
+    per_page: int = 100,
+    omit_revoked: bool = False,
+    omit_non_human: bool = False,
+):
+    async with logged_core_factory(config, device) as core:
+        user_info_tab, nb = await core.find_humans(
+            query, page, per_page, omit_revoked, omit_non_human
+        )
+    for user in user_info_tab:
+        click.echo(f"{user.human_handle} - UserID: {user.user_id}")
+    if not nb:
+        click.echo("No human found!")
+
+
+@click.command(short_help="search user id from email address")
+@click.argument("query", type=str)
+@core_config_and_device_options
```

Comment:
Would should add a `--include-revoked` flag, so that by default we only display non-revoked ids (the typical usecase for this comand being to retreive the current user_id for the given human handle)

```
$ parsec core human_find zack
Zack Snyder <zack@example.com> - UserID: acd17910ca5b4d3dbce4cc29a79fd2f8
$ parsec core human_find zack --include-rekoved
Zack Snyder <zack@example.com> - UserID: acd17910ca5b4d3dbce4cc29a79fd2f8
Zack Snyder <zack@example.com> - UserID: c7c7f53f7e8744d48ed3d106877bb20f (revoked)
```

Reply:
And I don't think the other params (such as the pagination and omit_non_human) are useful to expose: they don't fit the usecase (pagination) or are too complex to explain (omit_non_human)